app/models/approvals/individual.rb

```diff
@@ -0,0 +1,14 @@
+module Approvals
+  class Individual < Approval
```

Comment:
Please

* [ ] Add a comment explaining what this class is for
* [ ] Add a validation for `validates :user, presence: true` (assuming I understand it's purpose correctly)